crates/re_space_view/src/data_query_blueprint.rs

```
@@ -31,6 +32,33 @@ pub struct DataQueryBlueprint {
 
 impl DataQueryBlueprint {
     pub const OVERRIDES_PREFIX: &str = "overrides";
+
+    pub fn auto(space_view_class_name: SpaceViewClassName, space_path: &EntityPath) -> Self {
+        Self {
+            id: DataQueryId::random(),
+            space_view_class_name,
+            expressions: vec![space_path.to_string().into()].into(),
+        }
+    }
+
+    pub fn try_from_db(
+        path: &EntityPath,
+        blueprint_db: &StoreDb,
+        space_view_class_name: SpaceViewClassName,
+    ) -> Option<Self> {
+        let expressions = blueprint_db
+            .store()
+            .query_timeless_component::<QueryExpressions>(path)
+            .map(|c| c.value)?;
```

same here: silently swallowing deser errors I think?